source/ui/IDEAS/IDEASPersistentView.cpp

```diff
@@ -73,8 +73,11 @@ IDEASPersistentView::IDEASPersistentView(QWidget *parent) :
 
 	monoCrystal_ = new QLabel("Crystal");
 	monoCrystal_->setAlignment(Qt::AlignCenter);
-	monoEnergyRange_ = new QLabel(" eV - eV");
-	monoEnergyRange_->setAlignment(Qt::AlignCenter);
+
+	//*****************TEST: MONO TROUBLESHOOTING TOOL, WILL NEED IN FUTURE
```

Comment:
Once CID has time to investigate this odd mono behavior with me, I will likely need this troubleshooting tool.  It's handy to leave it here.